kickstarter/android-oss
# Bumping minSdkVersion to 23.

# What ❓
Branched off of #487 
I had to combine it with https://trello.com/c/LM9ObC1R/1232-remove-apicapabilities-checks-for-23 because of lint errors.
Updated LeakCanary to 1.6.3.
Removed resources in from api folders below 23 and into main files. I had to add a lot of ignores for `VectorPath` but we have some pretty inefficient vectors. 😬 

# How to QA? 🤔
Confirm the app no longer can be run on devices below 23 and LeakCanary successfully runs.

# Story 📖
[Trello](https://trello.com/c/xyhhnZd0/1231-set-minsdkversion-to-23)

# See 👀
<img width="626" alt="screen shot 2019-02-19 at 5 35 42 pm" src="https://user-images.githubusercontent.com/1289295/53052464-d23f8000-346c-11e9-99a3-773a61d50099.png">
